src/NHapi.Base/Util/Terser.cs

```diff
@@ -31,46 +31,63 @@
 
 namespace NHapi.Base.Util
 {
-   /// <summary> <p>Wraps a message to provide access to fields using a terse location
-   /// specification syntax.  For example: </p>
-   /// <p><code>terser.set("MSH-9-3", "ADT_A01");</code>  <br/>
-   /// can be used instead of <br/>
-   /// <code>message.getMSH().getMessageType().getMessageStructure().setValue("ADT_A01"); </code> </p>
-   /// <p>The syntax of a location spec is as follows: </p>
-   /// <p>location_spec: <code>segment_path_spec "-" field ["(" rep ")"] ["-" component ["-" subcomponent]] </code></p>
-   /// <p>... where rep, field, component, and subcomponent are integers (representing, respectively,
+   /// <summary>
+   /// Wraps a message to provide access to fields using a terse location specification syntax.
+   /// </summary>
+   /// <example>
+   /// <para>
+   /// For example:
+   /// <code>terser.set("MSH-9-3", "ADT_A01");</code> <br />
+   /// can be used instead of <br />
+   /// <code>message.getMSH().getMessageType().getMessageStructure().setValue("ADT_A01"); </code>

```

Comment:
most of the code examples are for hapi, these needs reviewing as part of documentation ticket.